samples/router/src/routes/Channel.tsx

```diff
@@ -8,6 +9,7 @@ export function Channel() {
   const createdAt = searchParams.get('createdAt')
   const numCreatedAt = Number(createdAt)
   const navigate = useNavigate()
+  useCustomRouting({ currentChannel: channelUrl })
```

Comment:
Does this `channelUrl` here actually mean `channel`?
The `currentChannel` should be a channel including the channel URL, instead of only URL